src/veh_interact.cpp

```diff
@@ -3128,7 +3128,7 @@ void veh_interact::complete_vehicle( Character &you )
             const inventory &inv = you.crafting_inventory();
             const requirement_data reqs = vpinfo.install_requirements();
             if( !reqs.can_make_with_inventory( inv, is_crafting_component ) ) {
-                add_msg( m_info, _( "You don't meet the requirements to install the %s." ), vpinfo.name() );
+                add_msg( m_info, _( "%1$s don't meet the requirements to remove the %2$s." ), you.disp_name() , vpi.name() );
```

Comment:
It's better to use [`add_msg_player_or_npc`](https://github.com/CleverRaven/Cataclysm-DDA/blob/f2c8c58036e1a48099055586e77fe175e9f4cca4/src/npc.h#L1264C14-L1267) so that the player is addressed with `you` and the NPC message uses the third person instead of the first. Same for the other changes.

Also, why change `install` to `remove`? This is actually installation, based on the switch condition above.

Reply:
Thank you for the quick response!
That sounds better indeed, and I wish I could answer to why I change install to remove, i guess i did some copy paste error. I will undo that.

Reply:
I have addressed your issues. Please resolve this conversation if you agree with me.